pkgs/by-name/le/lenmus/package.nix

```diff
@@ -23,23 +23,15 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "lenmus";
-  version = "6.0.1";
+  version = "6.0.1-unstable-2023-05-14";
```

Comment:
This date is wrong should probably be

```suggestion
  version = "6.0.1-unstable-2025-09-15";
```

Reply:
Indeed. Not sure what happened there. :(